### PR TITLE
JW-1077: refactor pkt0 send code

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

File: src/vnsw/agent/contrail/pkt0_interface.h [hunk 1]
```diff
@@ -15,6 +15,14 @@
 // pkt0 interface implementation of VrouterControlInterface
 class Pkt0Interface: public VrouterControlInterface {
 public:
+    #ifndef _WIN32
```
Comment (jacek-izykowski) [inline src/vnsw/agent/contrail/pkt0_interface.h:18]: https://google.github.io/styleguide/cppguide.html#Preprocessor_Directives

**Review by jacek-izykowski — APPROVED** [review]

**Review by jablonskim — APPROVED** [review]

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

**Comment by sodar** [comment]
Retest this please

**Pull request merged**